Application/EdFi.Ods.Profiles.Sample/Marker_EdFi_Ods_Profiles_Sample.cs

```diff
@@ -3,7 +3,9 @@
 // The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
 // See the LICENSE and NOTICES files in the project root for more information.
 
+using EdFi.Ods.Common.Extensibility;
+
 namespace EdFi.Ods.Profiles.Sample
 {
-    public interface Marker_EdFi_Ods_Profiles_Sample { }
+    public interface Marker_EdFi_Ods_Profiles_Sample : IPluginMarker { }
```

Comment:
Similar here, just marking the project won't affect its ability to build.

Reply:
I kept this file . Thanks